art/command_line.py

```diff
@@ -122,7 +143,7 @@ def download():
             _cache.get(filename)
         except KeyError:
             click.echo('* %s: %s => downloading...' % (entry['project'], entry['job_id']))
-            proj = gitlab.projects.get(entry['project'])
+            proj = gitlab.projects.get(entry['project'], lazy=True)
```

Comment:
Neat solution!

A brief comment would be useful to understand why a lazy manager is necessary here (not just an optimisation) -- when looking at the code outside of the git history context.

Reply:
yeah, I feel like `lazy` is always what you want. Too bad this kwarg isn't `eager=True`

Reply:
Expanded comments in 43cf7dba1ebf573b0114fb3e50cc68b61615c838